testing/src/main/scala/org/http4s/testing/ArbitraryInstances.scala

```diff
@@ -350,7 +364,7 @@ trait ArbitraryInstances {
   implicit val arbitraryContentLength: Arbitrary[`Content-Length`] =
     Arbitrary {
       for {
-        long <- arbitrary[Long] if long > 0L
+        long <- sized(n => choose(0L, n.toLong))
```

Comment:
The filter could cause this generator to fail on lists of headers.  This fix should be backported to 0.18.